bgv/encryptor.go

```diff
@@ -1,65 +0,0 @@
-package bgv
-
-import (
-	"github.com/tuneinsight/lattigo/v4/rlwe"
-)
-
-// Encryptor is an interface wrapping a rlwe.Encryptor, and modified
-// to accommodate for the BGV encryption by scaling the encryptions
-// of zero by the plaintext modulus before adding the plaintext.
-type Encryptor interface {
-	Encrypt(pt *Plaintext, ct *Ciphertext)
-	EncryptNew(pt *Plaintext) *Ciphertext
-	ShallowCopy() Encryptor
-	WithKey(key interface{}) Encryptor
-}
-
-type encryptor struct {
-	rlwe.Encryptor
-	params Parameters
-}
-
-// NewEncryptor instantiates a new Encryptor for the BGV scheme. The key argument can
-// be *rlwe.PublicKey, *rlwe.SecretKey or nil.
-func NewEncryptor(params Parameters, key interface{}) Encryptor {
```

Comment:
We should keep this method (with rlwe.Encryptor output)

Reply:
Also done